.github/workflows/deploy-feed-discovery-image.yaml

```diff
@@ -0,0 +1,27 @@
+name: Build and push feed-discovery service to ghcr
+
+on:
+  push:
+    branches:
+      - 'issue-1743'
```

Comment:
Let's isolate this to changes in the feed-discovery service:

```yaml
on:
  push:
    branches:
      - master
    paths:
      - 'src/api/feed-discovery/**'
```

cc @menghif who just did this for doc changes in https://github.com/Seneca-CDOT/telescope/commit/818ad8e257a66b288f77cbb6c4c4ca07066b4665